primitives/core/src/network.rs

```diff
@@ -90,6 +90,12 @@ pub enum Web3Network {
 	BitcoinP2wpkh,
 	#[codec(index = 13)]
 	BitcoinP2wsh,
+
+	// evm
```

Comment:
My review comes a bit late, better to put these two options above with `Ethereum/Bsc`.